src/docfx/build/page/BuildPage.cs

```diff
@@ -40,7 +40,7 @@ internal static class BuildPage
             (model.ContentGitUrl, model.OriginalContentGitUrl, model.OriginalContentGitUrlTemplate, model.Gitcommit) = context.ContributionProvider.GetGitUrls(file);
 
             List<Error> contributorErrors;
-            (contributorErrors, model.Author, model.Contributors, model.UpdatedAt) = await context.ContributionProvider.GetAuthorAndContributors(file, metadata.Author);
+            (contributorErrors, model.Author, model.Contributors, model.UpdatedAt) = await context.ContributionProvider.GetAuthorAndContributors(file, fileMetadata.Author);
```

Comment:
consider rename `model.Author` --> `model.AuthorInfo`

Reply:
There will be a lot of tests need to be change :(